docs/oauth2.md

```diff
@@ -538,13 +664,94 @@ In the following RAR example
 }
 ```
 
-if RabbitMQ node's `resource_server_id` is equal to `finance`, the plugin will compute the following sets of scopes:
+if RabbitMQ nodes `resource_server_id` is equal to `finance`, the plugin will compute the following sets of scopes:
 
 - `finance.read:primary-*/*/*`
 - `finance.write:primary-*/*/*`
 - `finance.configure:primary-*/*/*`
 - `finance.tag:administrator`
 
+
+
+## Advanced usage {#advanced-usage}
+
+### Use default OAuth 2.0 provider {#use-oauth-provider}
+
+As long you have one and only one OAuth 2.0 provider, you can skip this advanced usage although you can use it.
+
+Under the [basic usage](#configure-issuer) section, you configured the `issuer` url or maybe the `jwks_url` along with the TLS settings if needed. This advanced usage configures everything relative to the OAuth provider into a dedicated configuration.
+
+Here is a basic configuration as seen so far:
+
+```ini
+auth_oauth2.resource_server_id = rabbitmq-prod
+auth_oauth2.scope_prefix = rabbitmq.
+auth_oauth2.issuer = https://prodkeycloak:8080/realm/prod
+auth_oauth2.https.cacertfile = /opts/certs/prodcacert.pem
+```
+
+And the equivalent advanced configuration:
```

Comment:
I wouldn't start a sentence with "And" I would just say "The equivalent advanced configuration is: "